config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  get 'likes/create'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :create] do
+      resources :likes, only: [:create]
+      resources :comments, only: [:create]      
```

Comment:
- [ ] Please kindly refer to the comments above :arrow_up: We need two actions for creating a comment(:create, :new) hence a pair of routes here ` resources :comments, only: [:create, :new]   ` Form to create a comment should be found at `http://localhost:3000/users/1/posts/id/comments/new`

Reply:
Dear @FrederickMih ,
I would like to inform you that I have completed all the tasks outlined in the project requirements. With all due respect, based on my thorough assessment, I firmly believe that the comments you provided should be treated as suggested changes rather than mandatory modifications. I have taken into account the project specifications and executed the necessary actions accordingly.

Reply:
@kifle23, I suggested Rails best practices that permit us to create a scalable and quick debugging application. I'll kindly advice you to refactor your forms into the correct view files.

Reply:
Okay thanks @FrederickMih